.github/workflows/deploy-compodoc-to-github-pages.yml

```
@@ -0,0 +1,44 @@
+
+      - name: Generate Compodoc
+        run: pnpm run compodoc
+
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: documentation
+
+  deploy:
+    needs: build
+    uses: zhumeisongsong/shared-actions/.github/workflows/reusable-static-html-deploy-github-pages.yml@main
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)